src/app/views/docs/DocsShell/DocsShell.view.tsx

```diff
@@ -51,7 +51,10 @@ const DocsShellView: Component = () => {
         </Show>
       </aside>
       <main class={styles.main}>
-        <Suspense fallback={<Loader size="large" />}>
+        <Suspense
+          fallback={<Loader size="large" />}
+          children={undefined}
```

Comment:
this should probably be fixed - my IDE showed me an error here, but shouldn't the div below technically be considered to be children? 🤔